src/app/shared/figure/figure.component.html

```diff
@@ -0,0 +1,4 @@
+<figure class="figure" *ngFor="let item of items; let i = index">
+  <img src="{{ item.img }}" alt="" />
+  <figcaption>{{ item.description }}</figcaption>
+</figure>
```

Comment:
This component displayed an array of figures. Change component name to figures

Reply:
no fix